plugins/in_nginx_exporter_metrics/nginx.c

```
@@ -133,6 +133,13 @@ static int nginx_collect_stub_status(struct flb_input_instance *ins,
         goto client_error;
     }
 
+    /* User-Agent */
+    flb_http_add_header(client,
+                        FLB_HTTP_HEADER_USER_AGENT,
+                        sizeof(FLB_HTTP_HEADER_USER_AGENT) - 1,
+                        FLB_HTTP_HEADER_USER_AGENT_DEFAULT,
```

Cppcheck (2.19.0)

[error] There is an unknown macro here somewhere. Configuration is required. If FLB_VERSION_STR is a macro then please configure it.

(unknownMacro)